rosdep/python.yaml

```diff
@@ -3957,16 +3957,11 @@ python-rosinstall-generator:
   rhel:
     '7': [python2-rosinstall_generator]
   ubuntu:
-    lucid: [python-rosinstall-generator]
-    maverick: [python-rosinstall-generator]
-    natty: [python-rosinstall-generator]
-    oneiric: [python-rosinstall-generator]
-    precise: [python-rosinstall-generator]
-    quantal: [python-rosinstall-generator]
-    raring: [python-rosinstall-generator]
-    saucy: [python-rosinstall-generator]
-    trusty: [python-rosinstall-generator]
+    '*': [python-rosinstall-generator]
+    bionic_python3: [python3-rosinstall-generator]
+    focal_python3: [python3-rosinstall-generator]
     trusty_python3: [python3-rosinstall-generator]
+    xenial_python3: [python3-rosinstall-generator]
```

Comment:
I'm fine with switching this to use a `*`, but let's not add new `_python3` keys.  They were a failed experiment and we're trying to get rid of them.  If you'd like to add a Python 3 compatible key, please add a new `python3-rosinstall-generator` key instead.

Reply:
I see. My bad I also found [CONTRIBUTING.md](https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#python-rules) is updated for not mixing py3 with py2. Commit updated.

I'm not yet super familiar with Python3 situation esp. over other platforms than Ubuntu, so I've only filled in Ubuntu for Py3. Had I a bit more time I could dig into it, but there could also be a general guidance I assume (I can't find it e.g. in the CONTRIBUTING.md). Looking at a few other py3 entries the package names diverse...some with "3" but some without. If there's already an info that'd be helpful.